.stylelintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": ["stylelint-config-standard", "stylelint-prettier/recommended"],
+  "plugins": ["stylelint-order", "stylelint-scss"],
+  "rules": {
+    "order/properties-alphabetical-order": true,
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true,
+    "no-descending-specificity": [true, {"severity": "warning "}],
```

Comment:
This is turned on in  "stylelint-config-standard" by default.

I updated the files in Vanilla that were easy to fix, but we have quite some that are not that easy.
Nested selectors and many parent-child dependencies make it hard to reorder files in a way that specificity will be in order.

I turned it from error to warning, so we see it as a problem, but it doesn't fail the build.

BTW, Love how GH shows errors and warnings from linting on GitHub actions inline in the diff:
<img width="983" alt="Screenshot 2020-04-27 at 16 31 13" src="https://user-images.githubusercontent.com/83575/80383981-95f38500-88a4-11ea-91ee-3c8ef5b3f5ab.png">